src/clusterfuzz/_internal/issue_management/issue_tracker_utils.py

```diff
@@ -43,8 +44,16 @@ def _get_issue_tracker_project_name(testcase=None):
   return data_handler.get_issue_tracker_name(job_type)
 
 
-@memoize.wrap(memoize.FifoInMemory(256))
-def get_issue_tracker(project_name=None):
+def get_issue_tracker(project_name=None, is_appengine=True):
```

Comment:
Can we just use https://github.com/google/clusterfuzz/blob/961b23b21fd522b19eb66de9d5f3bf4d47b7b4be/src/clusterfuzz/_internal/system/environment.py#L718 instead of the argument?